CMakeLists.txt

```diff
@@ -1,15 +1,30 @@
 cmake_minimum_required(VERSION 3.20)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
C++11 is too low. If you need to specify the standard then it should be at least 17.